packages/rum-react/src/get-with-transaction.js

```diff
@@ -97,25 +98,7 @@ function getWithTransaction(apm) {
             return tr
           })
 
-          /**
```

Comment:
seems like you missed to carry over these comments to the utility.

Reply:
oops, it seems I was hungry and I ate the comment, gonna restore it